coverage/src/test/java/com/github/lizardev/xquery/saxon/coverage/trace/CoverageExpressionTest.java

```diff
@@ -0,0 +1,50 @@
+package com.github.lizardev.xquery.saxon.coverage.trace;
+
+import net.sf.saxon.expr.Expression;
+import net.sf.saxon.expr.instruct.TraceExpression;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.util.Objects;
+
+import static java.lang.String.format;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.BDDMockito.given;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CoverageExpressionTest {
+
+    @Mock private Expression expression;
+    @Mock private CoverageInstructionEventHandler handler;
+
+    @Test
+    public void shouldRemoveObjectsAddressInToString() {
+        CoverageExpression coverageExpression = new CoverageExpression(new TraceExpression(expression) {
```

Comment:
no sure if TraceExpression is needed as additional wrapper, in our code we have only CoverageExpression(expresion).